Cronos/cronos-evm-starter-via-eth/project.yaml

```diff
@@ -0,0 +1,55 @@
+specVersion: 1.0.0
+
+name: cronos-evm-starter-via-eth
+version: 0.0.1
+runner:
+  node:
+    name: "@subql/node-ethereum"
+    version: "*"
+  query:
+    name: "@subql/query"
+    version: "*"
+description: "This project can be use as a starting point for developing your Cosmos (Cronos) based SubQuery project"
+repository: "https://github.com/subquery/cosmos-subql-starter/tree/main/Cronos/cronos-evm-starter-via-eth"
+schema:
+  file: ./schema.graphql
+network:
+  chainId: "25"
+  # This endpoint must be a public non-pruned archive node
+  # Public nodes may be rate limited, which can affect indexing speed
+  # When developing your project we suggest getting a private API key
+  # Take a look here for some Cronos endpoints https://moonflow.solutions/rpc-servers
+  endpoint: "https://evm.cronos.org/"
+
+dataSources:
+  - kind: ethereum/Runtime
+    startBlock: 1474211
+    options:
+      abi: erc20
+      address: "0x5c7f8a570d578ed84e63fdfa7b1ee72deae1ae23" # Wrapped CRO
+    assets:
+      erc20:
+        file: "erc20.abi.json"
+    mapping:
+      file: "./dist/index.js"
+      handlers:
+        - handler: handleTransaction
+          kind: ethereum/TransactionHandler
+          filter:
+            ## The function can either be the function fragment or signature
+            ## https://docs.ethers.io/v5/api/utils/abi/fragments/#FunctionFragment
+            # function: '0x095ea7b3'
+            # function: '0x7ff36ab500000000000000000000000000000000000000000000000000000000'
+            function: approve(address spender, uint256 rawAmount)
+        - handler: handleLog
```

Comment:
Why only LogHandler? Is this simpler than using also ethereum/TransactionHandler?